pkg/daemon/daemon.go

```diff
@@ -598,9 +598,9 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 		if found {
 			switch profileClockType {
 			case "T-GM":
-				clockType = "GM"
+				clockType = event.GM
```

Comment:
this should default to T-GM, older T-GM profiles don't have  this value

Reply:
It does on 603 that was the issue.